.github/workflows/buildsCI.yaml

```diff
@@ -0,0 +1,327 @@
+name: Build_CI
+
+on:
+  push:
+    branches: 
+     - master
```

Comment:
Nir suggested we might want to change the branch names to work on all branches -- so it will trigger a build on our fork branches which are not master/development...

I.e.,
```
on:
  push:
    branches: ['**']
  pull_request:
    branches:  ['**']
```

Will it work?

Reply:
yes, it should work